Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -842,15 +910,15 @@ namespace Multiplayer
         {
             AZLOG_INFO("New incoming connection from remote address: %s", connection->GetRemoteAddress().GetString().c_str())
 
-            MultiplayerAgentDatum datum;
+                MultiplayerAgentDatum datum;
```

Comment:
nitpick: indentation is also off here due to the AZLOG_INFO on the previous line

Reply:
Fixed